xarray/core/types.py

```diff
@@ -114,6 +114,8 @@
 CoarsenBoundaryOptions = Literal["exact", "trim", "pad"]
 SideOptions = Literal["left", "right"]
 
+MPLHueStyleOptions = Literal["continuous", "discrete", None]
```

Comment:
```suggestion
HueStyleOptions = Literal["continuous", "discrete", None]
```

minor: there's no mpl connection here?